docs/_providers/axfrddns.md

```diff
@@ -0,0 +1,130 @@
+---
+name: AXFRDDNS
+title: AXFR+DDNS Provider
+layout: default
+jsId: AXFRDDNS
+---
+# AXFR+DDNS Provider
+
+This provider is able to work with any authoritative DNS server
+accepting AXFR requests (RFC5936) and Dynamic Update (RFC2136).
+
+It has been tested with [BIND](https://www.isc.org/bind/),
+[Knot](https://www.knot-dns.cz/), and
+[Yadifa](https://www.yadifa.eu/home).
+
+## Configuration
+
+### Authentication
+
+The AXFR+DDNS provider might work without anything in `creds.json` if
+the primary master of the zone accepts transfers and updates without
+TSIG authentication. But for non widely-open server, the
+authentication keys should be provided in `creds.json`.
```

Comment:
Suggested rewrite: (Warning: I'm not an expert in how AXFR and DDNS work. I took many guesses when writing this.  Please correct any assumptions.)

Authentication information is included in the `creds.json` entry for the provider:

* `transfer-key`: If this exists, the value is used to authenticate AXFR transfers.
* `update-key`: If this exists, the value is used to authenticate DDNS updates.

If either key is missing, DNSControl defaults to IP-based ACL authentication for that function.  Including both key is the most secure option.  Omitting both keys defaults to IP-based ACLs for all operations, which is the least secure option.